modules/Microsoft.RecoveryServices/vaults/.test/common/deploy.test.bicep

```diff
@@ -58,6 +58,7 @@ module testDeployment '../../deploy.bicep' = {
   scope: resourceGroup
   name: '${uniqueString(deployment().name)}-test-${serviceShort}'
   params: {
+    publicNetworkAccess: 'Enabled'
```

Comment:
Since the updated implementation, we should remove this setting. `publicNetworkAccess` will be disabled by default due to the implementation of private endpoints 
```suggestion
```